src/sdk/noobaa_s3_client/noobaa_s3_client.js

```diff
@@ -105,9 +105,35 @@ function check_error_code(err, code) {
     return err.code === code || err.Code === code;
 }
 
+/**
+ * When using aws sdk v3 we get the error object with values that start with an uppercase instead of lowercase
```

Comment:
I don't think it is the general case, it is only in the "Code" instead of "code" as far as I remember.
Maybe you can "LMLM err"?

I think that the comments might be helpful - but there is no actual case that we can use it (no other property with uppercase).

Reply:
All of the values are starting with an uppercase.

Reply:
```javascript
Feb-26 9:29:56.089 [BGWorkers/10894]    [L0] core.server.bg_services.namespace_monitor:: LMLM run_update_issues_report err NoSuchBucket: The specified bucket does not exist
    at throwDefaultError (/root/node_modules/noobaa-core/node_modules/@smithy/smithy-client/dist-cjs/default-error-handler.js:8:22)
    at /root/node_modules/noobaa-core/node_modules/@smithy/smithy-client/dist-cjs/default-error-handler.js:18:39
    at de_DeleteObjectTaggingCommandError (/root/node_modules/noobaa-core/node_modules/@aws-sdk/client-s3/dist-cjs/protocols/Aws_restXml.js:3713:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /root/node_modules/noobaa-core/node_modules/@smithy/middleware-serde/dist-cjs/deserializerMiddleware.js:7:24
    at async /root/node_modules/noobaa-core/node_modules/@aws-sdk/middleware-signing/dist-cjs/awsAuthMiddleware.js:30:20
    at async /root/node_modules/noobaa-core/node_modules/@smithy/middleware-retry/dist-cjs/retryMiddleware.js:27:46
    at async /root/node_modules/noobaa-core/node_modules/@aws-sdk/middleware-sdk-s3/dist-cjs/region-redirect-endpoint-middleware.js:14:24
    at async /root/node_modules/noobaa-core/node_modules/@aws-sdk/middleware-sdk-s3/dist-cjs/region-redirect-middleware.js:9:20
    at async /root/node_modules/noobaa-core/node_modules/@aws-sdk/middleware-logger/dist-cjs/loggerMiddleware.js:7:26 {
  '$fault': 'client',
  '$metadata': { httpStatusCode: 404, requestId: '6Y6QMAE6WCKHZ3B3', extendedRequestId: 'iBWbYEiLP9kR/k6FRZGCugA6Lt9IEwkn6q8xpGr9i6GOAFmVcKHJOp1oGmPcaEQF1o6C/sOLOlQ=', cfId: undefined, attempts: 1, totalRetryDelay: 0 },
  Code: 'NoSuchBucket',
  BucketName: 'liran-test-remove',
  RequestId: '6Y6QMAE6WCKHZ3B3',
  HostId: 'iBWbYEiLP9kR/k6FRZGCugA6Lt9IEwkn6q8xpGr9i6GOAFmVcKHJOp1oGmPcaEQF1o6C/sOLOlQ=',
  code: 'NoSuchBucket'
}
```